rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -716,4 +717,12 @@ public long getGasEstimationCap() {
     public DbKind databaseKind() {
         return DbKind.ofName(configFromFiles.getString(KeyValueDataSource.KEYVALUE_DATASOURCE_PROP_NAME));
     }
+
+    public double getPercentageOfPeersToConsiderInRandomSelection() {
+        if (!configFromFiles.hasPath(PROPERTY_PERCENTAGE_OF_PEERS_TO_CONSIDER_IN_RND_SELECTION)) {
```

Comment:
debugging stuff? we don't need this, right?

Reply:
Renamed